src/instructlab/configuration.py

```diff
@@ -216,10 +216,7 @@ class _generate(BaseModel):
         default=DEFAULTS.SDG_PIPELINE,
         description="Data generation pipeline to use. Available: 'simple', 'full', or a valid path to a directory of pipeline workflow YAML files. Note that 'full' requires a larger teacher model, Mixtral-8x7b.",
     )
-    model: StrictStr = Field(
-        default_factory=lambda: DEFAULTS.DEFAULT_MODEL,
-        description="Teacher model that will be used to synthetically generate training data.",
-    )
+    teacher: _serve = Field(default_factory=_serve, description="Teacher configuration")
```

Comment:
Instead of removing it right away, why not deprecate it first? So the users know about the flag duplication.